tests/test_commands.py

```diff
@@ -538,14 +538,14 @@ def test_url(self, url="test.com", domain="test.com"):
             domain,
             self.find_in_file(
                 Path(self.proj_mod_path, "spiders", "test_name.py"),
-                r"allowed_domains\s*=\s*\[\'(.+)\'\]",
+                r"allowed_domains\s*=\s*\[['\"](.+)['\"]\]",
             ).group(1),
         )
         self.assertEqual(
             f"http://{domain}/",
             self.find_in_file(
                 Path(self.proj_mod_path, "spiders", "test_name.py"),
-                r"start_urls\s*=\s*\[\'(.+)\'\]",
+                r"start_urls\s*=\s*\[['\"](.+)['\"]\]",
```

Comment:
Looks like something accidental happened with this file.

Reply:
No that was done intentionally... The regex now will detect single and double quotes.  The test fails otherwise, because it only tests for single quotes.

Reply:
Oh, right!!! My bad.